spec/server/process_spec.rb

```diff
@@ -662,4 +662,15 @@
       end
     end
   end
+
+  it "can schedule checks using cron" do
+    async_wrapper do
+      check = check_template
+      check[:cron] = "* * * * *"
+      create_check_request = @server.get_check_request_block(check)
+      @server.schedule_cron(check, &create_check_request)
+      expect(@server.instance_variable_get(:@timers).length == 1)
```

Comment:
You'll get better error messages with:

```ruby
expect(@server.instance_variable_get(:@timers).size).to eq(1)
```

Which'd return something like: "Expected expression to return 1, got 5" instead of "Expected expression to be true"